frontend/src/components/widgets/CameraInput/CameraInput.tsx

```diff
@@ -332,7 +350,14 @@ class CameraInput extends React.PureComponent<Props, State> {
           <img
             src={this.state.imgSrc}
             alt="Screenshot"
-            style={{ objectFit: "contain" }}
+            style={{
+              objectFit: "contain",
+              opacity:
+                this.state.shutter || !this.state.minShutterEffectPassed
+                  ? "50%"
+                  : "100%",
+              borderRadius: `.25rem .25rem 0 0`,
```

Comment:
@willhuang1997 if we have corresponding values in the theme, could you please use that values here too

Reply:
I'm getting an invalid hook usage when I import theme into here. How would I use theme in here?

Reply:
I think we can remove `borderRadius` here at all since it already exists on `StyledBox` that described in `styled-components.ts`

Reply:
if you go to https://share.streamlit.io/kajarenc/streamlit-camera-image-input/main/app.py, you can actually see that the image doesn't have rounded corners!